openid-caep-specification-1_0.txt

```diff
@@ -703,6 +713,45 @@ Cappalli & Tulshibagwale     Standards Track                   [Page 12]
       Figure 10: Example: Assurance Level Increase - Simple Subject +
                               optional claims
 
+
+
+
+
+
+
+
+
+
+
+
+
+Cappalli & Tulshibagwale     Standards Track                   [Page 13]
+
+                                CAEP-Spec                 September 2021
+
+
+   {
+       "iss": "https://idp.example.com/3456789/",
+       "jti": "07efd930f0977e4fcc1149a733ce7f78",
+       "iat": 1615305159,
+       "aud": "https://sp.example2.net/caep",
+       "events": {
+           "https://schemas.openid.net/secevent/caep/event-type/assurance-level-change": {
+               "subject": {
+                   "format": "iss_sub",
+                   "iss": "https://idp.example.com/3456789/",
+                   "sub": "jane.smith@example.com"
+               },
+               "namespace": "Retinal Scan",
```

Comment:
We should use an example that is from an actual registry since that is the primary use case.

Suggest we use NIST-AAL
```
...
"namespace": "NIST-IAL",
"current_level": "IAL2",
...
```

Reply:
This example is to show a "custom" assurance level, not one defined in the spec. NIST-IAL and NIST-AAL are defined in the spec.